### PR TITLE
Add scikit-learn

### PR DESCRIPTION
**Pull request by rth**
Scikit-learn 0.20 is the latest release to support Python 2.7 (see https://github.com/scikit-learn/scikit-learn/issues/11115 for more details).

scikit-learn 0.20-rc1 was released a few weeks ago, with the final release planned in the coming days. Assuming an (optimistic) 6 month release schedule, this means that the first python 3 only version will happen 6 month from now. The dates are somewhat approximate, but I think it's good enough.

### CODE REVIEW

File: site.js [hunk 1]
```diff
@@ -124,6 +124,10 @@ $(document).ready(function (){
       {content: 'Python 2 & 3', start: '2011-10-10', end:'2018-12-31', py2:true},
       {content: 'Python 3 only', start: '2019-01-01', end:'2021-12-16'},
     ],
+    'scikit-learn':[
+      {content: 'Python 2 & 3', start: '2010-03-25', end:'2019-03-18', py2:true},
```
Comment (rth) [inline site.js]: Using the 0.1 release as the starting point, though this shouldn't matter much.

**Comment by Carreau** [comment]
Awesome ! 

Thanks !

Feel free to ping any of us when you transition, read the [practicalities](http://python3statement.org/practicalities/) section to make sure to use python_requires correctly (to not break for Python 2 users, and feel free to ping here if you need a pair of eyes on the Pr that drop 2.7.

**Comment by rth** [comment]
Thanks a lot for the review and suggestions !

**Comment by takluyver** [comment]
cc @amueller @GaelVaroquax @ogrisel @jnothman - we generally ask project maintainers to agree on signing this statement, even if the project's release schedule is compatible with it. Some projects (like Django) have made Python-3-only releases but chosen not to sign the statement.

In this case, scikit-learn already got added. Obviously we hope you'll leave it there - the statement is a more persuasive argument with more projects on board - but if you decide you'd rather not, please let us know.

**Comment by takluyver** [comment]
Extra cc @GaelVaroquaux because I misspelled his handle above.

**Comment by amueller** [comment]
I think @rth is in a fine position to make the call ;) But for the record I think it's good to sign.
The estimate for the release timeline is a bit optimistic though ;)

**Comment by GaelVaroquaux** [comment]
+1 on what @amueller said. But it's fine to have a somewhat optimistic release timeline here.

**Comment by ogrisel** [comment]
+1 as well.

**Comment by takluyver** [comment]
Thanks everyone. :-)

**Pull request merged**